keras_cv/layers/feature_pyramid.py

```diff
@@ -66,6 +66,12 @@ class FeaturePyramid(tf.keras.layers.Layer):
             with feature inputs and merged result from upstream levels. Default to None,
             and a `keras.Conv2D` layer with kernel 3x3 will be created for each pyramid
             level.
+        num_extra_level: int, extra level of pyramid on top of existing `max_level`
```

Comment:
I've noticed we have some inconsistency in our API about how we handle parameters that represent a count of something

For example, here (and with `num_channels` above), we use the `num_` prefix, but for `classes` we exclude the `num_` prefix (and in the Grayscale preprocessing layer we use `output_channels`, not `num_output_channels`).

How would you feel about standardizing this to always exclude the `num_` prefix? Then we could call this `extra_levels` and rename `num_channels` to `channels`

Reply:
Good question. I think the "num_" prefix is indicating that the param is a number (usually a positive int). If we are fine with using docstring/types to provide the type information, then I think its totally make sense to remove this prefix. @tanzhenyu  and @LukeWood

Reply:
IMHO I think `num_classes` is better...I do imagine that there will be some APIs -- especially visualization related APIs that would require `classes` to be a dict between index and the class name.

Reply:
I agree with @tanzhenyu that num_classes is better, but @fchollet and I think a few others agreed classes is better.  The group is about 50/50 on that one

Reply:
Zhenyu’s point regarding visualization tools needing a dictionary for classes  is a really good point.